### PR TITLE
Update README & Bump actions/checkout version to V3

### PR DESCRIPTION
**Pull request by amos-kibet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +127,7 @@ steps:
 
   - name: Deploy app
     id: deploy
-    uses: amos-kibet/fly-pr-review-apps@v1
+    uses: optimumBA/fly-pr-review-apps@main
```
Comment (almirsarajcic) [inline README.md]:
```suggestion
    uses: optimumBA/fly-preview-apps@main
```

Fix it in the other places as well.

File: README.md [hunk 2]
```diff
@@ -36,8 +35,8 @@ on:
 
 env:
   FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
-  FLY_REGION: iad
-  FLY_ORG: personal
+  FLY_REGION: <put your desired region>
+  FLY_ORG: <put your fly org name>
```
Comment (almirsarajcic) [inline README.md]: I think it's better to show an example instead of explaining what to put. Please revert this two lines.

**Review by almirsarajcic — CHANGES_REQUESTED** [review]
Can you please undo changes to LICENSE file?

**Comment by amos-kibet** [comment]
Hopefully, I got it right to revert LICENSE.

**Review by almirsarajcic — APPROVED** [review]

**Pull request merged**